### PR TITLE
Zombies can see Initial Infected

### PR DESCRIPTION
**Pull request by psykana**
## About the PR
Zombies can now see initial infected.

## Why / Balance
Many times I've seen Initial Infected's sabotage plans get ruined by another II who's starting to bite people 5 minutes into the match, making the former fight both the living and the agressive turned.
This should improve gameplay as Initial Infected and zombies in general, as they will cooperate better now.

This is kind of controversial in terms of lore, but both options can be justified:
- On one hand, zombies are completely braindead and just attack all living.
- On the other hand, Initial Infected are Syndicate bioterrorists who have/keep the virus since the beginning on the match, and zombies can sense it somehow, just like they can sense each other.

## Technical details
Only prototype change.

## Media
image

## Requirements
- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
None

**Changelog**
:cl:
- add: Zombies can now see initial infected

### CODE REVIEW

**Review by slarticodefast — APPROVED** [review]
I assume it has been this way before because zombies are supposed to be mindless and attack everyone living in sight. But it practice I agree with you and I had similar situations happen before. When two initial infected try to coordinate to turn and attack at the same time losing the icon makes it hard to keep track of your former ally. And in uncoordinated cases it can ruin the round for the initial infected when they get randomly attacked by another zombie. Hiding among the living and striking at a good opportunity makes the roundflow more interesting in my opinion.

**Comment by 2013HORSEMEATSCANDAL** [comment]
I'd justify it by saying that the II are already infected, their symptoms just aren't noticeable to humans.

**Comment by chromiumboy** [comment]
I agree with the change personally, but it probably should get approval from the admin team first

**Comment by UbaserB** [comment]
>On the other hand, Initial Infected are Syndicate bioterrorists who have/keep the virus since the beginning on the match, and zombies can sense it somehow, just like they can sense each other.

Might be cool to let zombies also sense who is already infected and not be allowed to attack those people

**Comment by thebadman4662** [comment]
> > Might be cool to let zombies also sense who is already infected and not be allowed to attack those people
> 
> Allowing zombies to know who's infected would be cool, but just straight up not allowing them to attack someone infected would be bad. Infections take quite awhile to actually do anything, which is good, since it gives survivors a chance to get a cure, so I don't think it should be changed. If I just straight up could not fight the person shooting me because they already got infected I would be pretty mad. There's also cases with you just getting mauled by a horde and you shouldn't be able to just run out alive after one of the bites trigger an infection and suddenly you're immune to zombies.

Pretty sure by infected they meant IIs antags, not crew with zombie DoT.

**Comment by slarticodefast** [comment]
Hey, this was brought up for internal discussion and for rule clarity reasons we have one small change request before this can be merged:
Could you update the character info text for zombies so they know not to attack intial infected?
grafik
Something like "Work together with the other zombies and initial infected to overtake the station" should do.
See `zombie.ftl` for that translation string.

**Comment by lzk228** [comment]
then maybe make zombies not to bite IIs?

**Review by slarticodefast — APPROVED** [review]
Thanks!

**Comment by psykana** [comment]
Hi @slarticodefast, what's stopping this from getting merged?

**Comment by lzk228** [comment]
> Hi @slarticodefast, what's stopping this from getting merged?

double approval requirement

**Comment by slarticodefast** [comment]
The code has been approved by me and the admins gave the conceptual ok. Acoording to maintainer policy another maintainer will have to double check the PR before merging. I can try and annoy a few people :)

**Pull request merged**
